src/components/AddJournal.js

```diff
@@ -0,0 +1,122 @@
+import { React, useState } from "react";
+import { Row, Col, Form, Button } from "react-bootstrap";
+
+const AddJournal = () => {
+  const [title, setTitle] = useState("");
+  const [url, setUrl] = useState("");
+  const [issn, setIssn] = useState("");
+  const [rating, setRating] = useState("");
+  const [date, setDate] = useState("");
+  const [policyTitle, setPolicyTitle] = useState("");
+  const [firstYear, setFirstYear] = useState("");
+  const [lastYear, setLastYear] = useState("");
+  const [policyType, setPolicyType] = useState("");
+  const [domain, setDomain] = useState("");
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    const policies = {
+      title: policyTitle,
+      first_year: firstYear,
+      last_year: lastYear,
+      type: policyType,
+    };
+    const journal = { title, url, issn, rating, date, policies, domain };
+
+    fetch("https://journal-policy-tracker.herokuapp.com/api/journals", {
```

Comment:
Small request: This is fine for early dev, but now that we have multiple API calls in multiple places, let's try to move the URLs we're fetching from to a single config file. That way if we have to change hosts, we'll be able to switch quickly and only edit one single line.

Reply:
Noted